lib/ixml.js

```diff
@@ -107,6 +107,7 @@ const I_XML_ATTR_KEY_DB = "db";
 const I_XML_ATTR_KEY_USERID = "uid";
 const I_XML_ATTR_KEY_PASSWORD = "pwd";
 const I_XML_ATTR_KEY_IO = "io";
+const I_XML_ATTR_KEY_BY = "by";

```

Comment:
Personally, I'd get rid of all these `I_XML_ATTR`* values and just use the literal value. They are not configurable or going to change and just result in more typing and obfuscation.

I'm not saying to remove them as part of this PR, but I wouldn't add any more.

@abmusse @markdirish Thoughts?

Reply:
@kadler  seems reasonable to me many times when I have to scroll back to find the definition of the constant. 

We could add this to our v1.0 project goals.

Reply:
I just continued the technique for the constant.  Agreed it would be nice to eliminate moving forward..